package.json

```diff
@@ -25,7 +25,15 @@
   "homepage": "https://github.com/PrismarineJS/prismarine-template#readme",
   "devDependencies": {
     "jest": "^27.0.4",
-    "prismarine-template": "file:.",
```

Comment:
maybe add "xboxlive-auth" as a self dep so it would be easier to reference this package in tests? idk if this pr will implement tests though